cmd/flags.go

```diff
@@ -165,6 +165,11 @@ func addV1CompatibleFlag(fs *pflag.FlagSet, v1Compatible *bool, value bool) {
 	fs.BoolVar(v1Compatible, "v1-compatible", value, "opt-in to OPA features and behaviors that are enabled by default in OPA v1.0")
 }
 
+func addReadAstValuesFromStoreFlag(fs *pflag.FlagSet, readAstValuesFromStore *bool, value bool) {
+	// FIXME: naming?
```

Comment:
The concern I have here is that without the right context, `--read-ast-values` tells you almost nothing about what this flag does. 
Changing it to `--read-ast-values-from-store` would give some more context; but it's almost misleading, since _reading AST values_ is more of a side effect of what we're actually doing: _eagerly converting stored data to AST_.

Reply:
`--read-ast-values-from-store` seems better and if we add a good description that could help.

```
eagerly converting stored data to AST.
```

If we always stored AST then it wouldn't, no? But that has other consequences.